lib/ui/sign_transaction/interactor/data/transaction_action_data.dart

```diff
@@ -20,6 +22,7 @@ class TransactionDetailsData {
     return TransactionDetailsData(
       signingTitle: 'From ${data.esr.actions.first.account}',
       expirationTime: expiration,
+      isFreeTransaction: data.transaction.hasFreeCpuAction,
```

Comment:
@n13  Is this how we check if to show the Free Transaction in the UI or not?

Reply:
Yes!